README.md

```diff
@@ -106,7 +106,8 @@ There are also some Dragonfly-specific arguments:
  * `dbnum`: Maximum number of supported databases for `select`.
  * `cache_mode`: See the [novel cache design](#novel-cache-design) section below.
  * `hz`: Key expiry evaluation frequency (`default: 100`). Lower frequency uses less CPU when idle at the expense of a slower eviction rate.
- * `save_schedule`: Glob spec for the UTC to save a snapshot in HH:MM (24h time) format (`default: ""`).
+ * `snapshot_cron`: Cron schedule expression for automatic backup snapshots using standard cron syntax with the granularity of minutes (`default: ""`).
```

Comment:
it's not straightforward so I think it's a great place to add some examples with the explanation.

Reply:
Added examples, linked to our "Saving Backups" doc. 😃